wiki-webui/src/main/java/org/exoplatform/wiki/webui/control/action/ExportAsPDFActionComponent.java

```diff
@@ -138,6 +138,10 @@ private String htmlToxhtmlAndBase64Images(Page page) throws WikiException {
           String src = element.attr("src");
           String imgName = src.substring(src.lastIndexOf("/") + 1);
           Attachment imageAttachment = wikiService.getAttachmentOfPageByName(imgName,page,true);
+          if(imageAttachment == null) {
+            Page parentPage = wikiService.getParentPageOf(page);
+            imageAttachment = wikiService.getAttachmentOfPageByName(imgName, parentPage, true);
+          }
           if(imageAttachment.getMimeType().startsWith("image")){
```

Comment:
please add another check here for imageAttachment != null and imageAttachment.getMimeType() != null (if it can be null)

Reply:
Done